### DISCUSSION TITLE
I managed to run NDI plugin on Apple Silicon OBS 28 version! Sort of....

### DISCUSSION THREAD

**Discussion by nunowonder** [DistroAV/DistroAV#787] [General]
The main problem is: when you try to open OBS28 Apple Silicon (Arm64) version after installing the 4.10 version of NDI plugin and installing also NDI runtime 4.5.1, the NDI plugin just don't open. To go around the problem, I have the latest NDI tools installed on my Mac, so if I go to NDI Video Monitor content package, inside Framework folder, I just copy that libndi_advanced.dylib and paste it to folder /usr/local/lib, and inside there is a file called libndi.4.dylib. Then you just rename the one that came from NDI Video Monitor,  to this name (so you just erase this file or change its name or something). Then, NDI runs on OBS 28 Apple Silicon...
BUT!... of course there is a BUT... the sound is terrible...

If we use the 5.0 version of the plugin will it work? I need that so badly...

**Comment by SouthWestie** [comment]
Did you ever get the NDI runtime not found error when opening OBS?

**Reply by nunowonder** [reply]
just do what I did, all the steps. But it's not worth it. The OBS plugin makes lot of distortion noises

**Reply by centineo** [reply]
Hi,

I did the same as you, and the noise comes from the source, putting side by side NDI monitor / OBS programs projector 
there is some noise but it comes prior to pass throughout the OBS 
Maybe id NDI problem working whit apple M1

**Reply by nunowonder** [reply]
If it was, you couldn’t use NDI at all. But if I use NDI plug-in prior to this new version,
Everything is ok.

**Comment by nunowonder** [comment]
I remembered that I found the solution (well, an alternative for the perfect way, that will only be available as soon as this plugin is ok for M1 computers) so here it is:
You should install obs-ndi-5.0.0-macos-arm64.pkg (I know you won't see this version but just search for it on google or something).
Also, install the latest NDI tools.
Let me warn you that the audio right channel of the plugin will be distorting... that's something that will be corrected hopefully in the next release. 
You shall use NDI Virtual Input software, then capture what you want on that program, and then in OBS just add a source of audio Input Capture, select the NDI Audio, and mute the NDI5 plugin. 
Hope will work for you, I know its not perfect but it's what we have (by "we", I mean M1 users)
